### ISSUE TITLE
Add verifyEmail to STATE_PAT

### ISSUE THREAD

**Issue by dalgard** [meteor-useraccounts/core#316]
I am using this state `verifyEmail` in my site's login solution and would like to be able to retrieve a title from the translation file. Please insert `verifyEmail: Match.Optional(String)` after this line:

https://github.com/meteor-useraccounts/core/blob/84ff413f73948e127f1dbf2289e5c6925a083091/lib/core.js#L13

The default string `verifyEmail: "verifyEmail"` should also be added after this line:

https://github.com/meteor-useraccounts/core/blob/84ff413f73948e127f1dbf2289e5c6925a083091/lib/client.js#L87

**Comment by splendido** [comment]
I can add what you're asking without problems but:

* there's no `verifyEmail` within accounts-t9n keys (you could add it on your own though...)
* the form would result empty...

How are you using it?

**Comment by splendido** [comment]
I went for a neutral solution, but you should get the title with:

```javascript
AccountsTemplates.configure({
    texts: {
      title: {
        verifyEmail: "verifyEmail",
      }
    }
});
```

Now you can also *hide* a title by setting it's corresponding text to an empty string ;-)

**Comment by dalgard** [comment]
Awesome, thanks! Did you notice that I edited my message to include adding `verifyEmail` to the default configuration?

**Comment by splendido** [comment]
Yep!
But I put it as an ampy string by default to keep the title hidden as default ;-)

**Comment by dalgard** [comment]
Sorry – I can see in your code. Thanks :+1:

**Comment by splendido** [comment]
will be shipped with v1.9.0 in some next future! :-D

tnx
